themes/default/content/docs/intro/console/stack-permissions.md

```diff
@@ -15,7 +15,16 @@ The Pulumi Console provides fine-grained access controls for stacks.
 A user's stack permissions is first based on their role within the containing
```

Comment:
thoughts on moving stack permissions page content to the stacks and project page?